views/includes/check-your-answers/trusts.html

```diff
@@ -1,9 +1,54 @@
+{% import "includes/date-macros.html" as dateMacros %}
 <h3 class="govuk-heading-m">Trust {{ trust_count }}</h3>
 
 {% set trustBeneficialOwnersHtml %}
   {% include "includes/check-your-answers/trust-beneficial-owners.html" %}
 {% endset %}
-
+{% if pageParams.isTrustFeatureEnabled %}
+{% set trustCreationDate = dateMacros.formatDate(trust.creation_date_day, trust.creation_date_month, trust.creation_date_year) %}
```

Comment:
could you indent the code (one step) inside the {% if pageParams.isTrustFeatureEnabled %}
and inside the else